src/install.rs

```diff
@@ -293,6 +293,12 @@ pub fn bls_entry_delete_and_append_kargs(
     delete_args: Option<&Vec<String>>,
     append_args: Option<&Vec<String>>,
 ) -> Result<Option<String>> {
+    if (delete_args.is_none() || delete_args.unwrap().is_empty())
+        && (append_args.is_none() || append_args.unwrap().is_empty())
```

Comment:
It seems like the `Option<>` wrappers are redundant in the function signature?

Reply:
You mean, canonicalize `None` to empty `Vec`s? Yeah, I think we could do that with an `unwrap_or_else()` in the two `cmdline.rs` code which deal with this.

Let me do that in a follow-up.

Reply:
Did this in this PR now!